agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/perfcounter/JmxDataFetcher.java

```diff
@@ -44,7 +45,7 @@ public static Map<String, Collection<Object>> fetch(
     MBeanServer server = ManagementFactory.getPlatformMBeanServer();
     Set<ObjectName> objects = server.queryNames(new ObjectName(objectName), null);
     if (objects.isEmpty()) {
-      String errorMsg = String.format("Cannot find object name '%s'", objectName);
+      String errorMsg = String.format(Locale.ROOT, "Cannot find object name '%s'", objectName);
```

Comment:
just fyi, I usually replace String.format with string concatenation, similar to upstream: https://github.com/open-telemetry/opentelemetry-java/pull/4887#discussion_r1005841047